Submitted By: linuxfromscratch DOT org
Date: 2020-11-19 
Initial Package Version: 1.18.3
Origin: http://www.linuxfromscratch.org/blfs/view/svn/postlfs/mitkrb.html
Description: Patch removes a test that is known to fail.

diff --color -Nur krb5-1.18.3.orig/src/plugins/kdb/db2/libdb2/test/run.test krb5-1.18.3/src/plugins/kdb/db2/libdb2/test/run.test
--- krb5-1.18.3.orig/src/plugins/kdb/db2/libdb2/test/run.test	2020-12-07 04:58:27.210999666 +0100
+++ krb5-1.18.3/src/plugins/kdb/db2/libdb2/test/run.test	2020-12-07 04:58:48.579210832 +0100
@@ -38,7 +38,7 @@
 	find $bindir -type f -exec test -r {} \; -print | head -100 > $BINFILES
 
 	if [ $# -eq 0 ]; then
-		for t in 1 2 3 4 5 6 7 8 9 10 11 12 13 20 40 41 50 60 61 62 63; do
+		for t in 1 2 3 4 5 6 7 8 9 10 11 13 20 40 41 50 60 61 62 63; do
 			test$t
 		done
 	else
